test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,36 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding Request with Custom Environment Variables
+	Given Imported Nodejs application "nodejs-rest-http-crud-custom-env-var" is running
+	* DB "db-demo-custom-env-var" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-custom-env-var
+            spec:
+                applicationSelector:
+                    resourceRef: nodejs-rest-http-crud-custom-env-var
+                    group: apps
+                    version: v1
+                    resource: deployments
+                backingServiceSelectors:
+                   - group: postgresql.baiju.dev
+                     version: v1alpha1
+                     kind: Database
+                     resourceRef: db-demo-custom-env-var
+                     id: postgresDB
+                customEnvVar:
+                   - name: TAGS
+                     value: '{{ .postgresDB.spec }}'
+                   - name: ARCHIVE_USERLABEL
+                     value: '{{ .postgresDB.spec.userLabels.archive }}'
+                   - name: SECONDARY_SECRETNAME
+                     value: '{{ .postgresDB.spec.secretName }}'
+            """
+        Then application should be re-deployed
+	And application should be connected to the DB "db-demo-custom-env-var"
+	And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-custom-env-var" should be changed to "True"
+	And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-custom-env-var" should be changed to "True"
```

Comment:
As mentioned above, we need to check the content of each custom env var as well.

I would suggest to split this scenario into multiple scenarios, demonstrating just single value type.

Reply:
Something along the lines of:

```feature
And key "TAGS_1" from Service Binding "binding-request-custom-env-var" secret should be "value from TAGS_1"
And key "TAGS_2" from Service Binding "binding-request-custom-env-var" secret should be "value from TAGS_2"
```